src/fuel-vm/instruction-set.md

```diff
@@ -2491,19 +2490,13 @@ Get [fields from the transaction](../tx-format/transaction.md).
 | `GTF_INPUT_COIN_OWNER`                    | `0x203` | Memory address of `tx.inputs[$rB].owner`                          |
 | `GTF_INPUT_COIN_AMOUNT`                   | `0x204` | `tx.inputs[$rB].amount`                                           |
 | `GTF_INPUT_COIN_ASSET_ID`                 | `0x205` | Memory address of `tx.inputs[$rB].asset_id`                       |
-| `GTF_INPUT_COIN_TX_POINTER`               | `0x206` | Memory address of `tx.inputs[$rB].txPointer`                      |
```

Comment:
weren't we going to use tx_pointer as a replacement for the maturity field? Why are we not setting it anymore?

cc @xgreenx

Reply:
Discussed on a call - the tx_pointer will be unreliable for predicates to use as a maturity check. The most sensible approach for now is to have predicates check the tx-wide maturity policy.